client/my-sites/earn/home.tsx

```diff
@@ -52,8 +52,7 @@ interface ConnectedProps {
 	siteId: number;
 	selectedSiteSlug: SiteSlug;
 	isFreePlan: boolean;
-	isJetpack: boolean;
-	isAtomicSite: boolean;
+	isJetpackNotAtomic: boolean;
```

Comment:
Picky, but I feel like `isNonAtomicJetpack` might read a little more easily at a glance.

Reply:
Updated to that! :)